src/store.cc

```diff
@@ -65,19 +64,18 @@ void MetadataStore::UpdateMetadata(const std::string& full_resource_name,
   }
   // Force value update. The repeated search is inefficient, but shouldn't
   // be a huge deal.
-  metadata_map_.erase(full_resource_name);
-  metadata_map_.emplace(full_resource_name, std::move(entry));
+  const MetadataKey metadata_key{entry.name, entry.version};
+  metadata_map_.erase(metadata_key);
+  metadata_map_.emplace(metadata_key, std::move(entry));
 }
 
-std::map<std::string, MetadataStore::Metadata>
-    MetadataStore::GetMetadataMap() const {
+std::vector<MetadataStore::Metadata> MetadataStore::GetMetadataList() const {
```

Comment:
Why not just `GetMetadata`?

Reply:
The naming I proposed would be consistent with `ReportMetadata` and `SendMetadata` in `MetadataReporter`. The fact that the `MetadataStore::Metadata` class represents a single object is not relevant either — this is about descriptive naming of the API elements. WDYT?

Reply:
Makes sense about being consistent with SendMetadata. Changed it to GetMetadata.